plugins/poll/assets/javascripts/controllers/poll-ui-builder.js.es6

```diff
@@ -2,13 +2,15 @@ import Controller from "@ember/controller";
 import {
   default as computed,
   observes
-} from "ember-addons/ember-computed-decorators";
+} from "discourse-common/utils/decorators";
 import EmberObject from "@ember/object";
 
 export default Controller.extend({
   regularPollType: "regular",
   numberPollType: "number",
   multiplePollType: "multiple",
+  pieTypeIdentifier: "pie",
```

Comment:
Sorry for the miscommunication here but if these values don't change they needn't be attached to every object. Instead what I'd recommend is creating `const PIE_TYPE = "pie";` outside of the declaration of this class and use that `const` for comparisons.